lib/components/viewers/stop-time-cell.tsx

```diff
@@ -109,7 +114,10 @@ const StopTimeCell = ({
           status: getTripStatus(realtime, stopTime.departureDelay, 30) as status
         })}
       >
-        {showCountdown ? (
+        {/* Not the cleanest boolean, but makes it very clear what we're doing
+        in all cases. */}
+        {(onlyShowCountdownForRealtime === true && realtime) ||
+        (onlyShowCountdownForRealtime === false && showCountdown) ? (
```

Comment:
Can this not be 
```suggestion
        {(onlyShowCountdownForRealtime && realtime) ||
        (!onlyShowCountdownForRealtime && showCountdown) ? (
```

Reply:
The explicit `true` helps us avoid false/null/undefined confusion